praw/models/reddit/modmail.py

```diff
@@ -70,7 +70,8 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
         """Construct an instance of the ModmailConversation object."""
         super(ModmailConversation, self).__init__(reddit, _data)
 
-        self.id = id  # pylint: disable=invalid-name
+        if id is not None:
```

Comment:
Can we add a unit test case that shows that this is necessary? E.g., it would fail if it were not in place?